### PR TITLE
Add new exercise: bank account

### PR DESCRIPTION
**Pull request by brendanmckeown**
Fixes issue [#767](https://github.com/exercism/javascript/issues/767)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -100,6 +100,17 @@
         "strings"
       ]
     },
+    {
+      "slug": "bank-account",
+      "uuid": "8beedf70-28b4-4cfb-ab53-309ee6f6aa78",
+      "core": false,
+      "unlocked_by": null,
```
Comment (SleeplessByte) [inline config.json]: Pick any exercise with the value `"core": true`. Something where you feel like this should be available after.

File: config.json [hunk 2]
```diff
@@ -100,6 +100,17 @@
...
+      "difficulty": 2,
```
Comment (SleeplessByte) [inline config.json]: I think this is too low.

File: exercises/bank-account/bank-account.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    throw new Error('Remove this statement and implement this function');
+  }
+
+  get balance() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+}
+
+export class ValueError extends Error {}
```
Comment (SleeplessByte) [inline exercises/bank-account/bank-account.js]: Can you add a constructor that sends to `super('A descriptive error message')` so the user better understands what a `ValueError` is?

**Comment by SleeplessByte** [comment]
Pinging @brendanmckeown

**Comment by SleeplessByte** [comment]
> package.json did not match for bank-account

Probably need to rebase/merge in master and then run the make sync script.

**Comment by TomPradat** [comment]
@brendanmckeown Any plans to finish this MR ?

**Comment by SleeplessByte** [comment]
Thanks for working on this. @TomPradat has done the last step in #888. Author is still you!

**Pull request closed without merging**